spec/integration_spec.rb

```diff
@@ -829,6 +830,14 @@ class SerializedDocument < ActiveRecord::Base
     expect(facets.size).to eq(1)
     expect(facets).to include(black)
   end
+
+  it 'searches with sorting' do
+    blue = Color.create!(name: 'blue', short_name: 'blu', hex: 0x0000FF)
+    black = Color.create!(name: 'black', short_name: 'bla', hex: 0x000000)
+    green = Color.create!(name: 'green', short_name: 'gre', hex: 0x00FF00)
+    facets = Color.search('*', { sort: ['name:asc'] })
+    expect(facets).to eq([black, blue, green])
```

Comment:
I always suggest this improvement, looks like a silly thing, but it helps a lot when we are reading the test!

ref: http://blog.plataformatec.com.br/2014/04/improve-your-test-readability-using-the-xunit-structure/

```suggestion

    facets = Color.search('*', { sort: ['name:asc'] })

    expect(facets).to eq([black, blue, green])
```

Reply:
Thanks for linking that article! I can definitely see the readability improvements by leveraging this approach.